docs/otterize-oss/errors-telemetry.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Errors telemetry
+---
+
+Otterize OSS components report anonymous errors telemetry that allows us to proactively identify and address issues. No personal or organizational identifying information is transmitted in these metrics: The information reported is limited to essential details necessary for effective issue resolution. You may opt out at any time through a single configuration flag.
+
+## What information is collected?
+
+Each error event includes error type, relevant code lines from the stack trace, and metadata on the OSS component instance identifiers, detailed in the table below. For a comprehensive understanding of the schema used for Bugsnag error reporting, please refer to the following link: [Bugsnag Error Reporting API](https://bugsnagerrorreportingapi.docs.apiary.io/#reference/0/notify/send-error-reports).
```

Comment:
Maybe mention something about how we use Bugsnag SDK to format the error report before referring to the bugsnag docs?